core/sql/regress/hive/EXPECTED003

```diff
@@ -138,6 +132,10 @@ P_PROMO_SK   P_PROMO_ID                 P_START_DATE_SK  P_END_DATE_SK  P_ITEM_S
 
 *** ERROR[8822] The statement was not prepared.
 
+*** WARNING[8597] Statement was automatically retried 1 time(s). Delay before each retry was 0 seconds. See next entry for the error that caused this retry.
```

Comment:
Do we want to print out these warnings when the retry failed for the same reason as the original error? Seems like unnecessary repetition.